.github/workflows/main.yml

```
@@ -0,0 +1,20 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Recommendation is to make sure things work on 18 and 20 these days, 16 will no longer receive any support in October.

```suggestion
        node-version: [20.x]
```

---

Updated [a31aa7f](https://github.com/credential-handler/chapi-playground-contexts-server/pull/1/commits/a31aa7f05c788f11d79885ab628ed8032b603467)